bitnami/wordpress/templates/deployment.yaml

```
@@ -58,8 +58,37 @@ spec:
       {{- if .Values.podSecurityContext.enabled }}
       securityContext: {{- omit .Values.podSecurityContext "enabled" | toYaml | nindent 8 }}
       {{- end }}
-      {{- if .Values.initContainers }}
-      initContainers: {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
+      {{- if or .Values.initContainers .Values.volumePermissions.enabled }}
+      initContainers:
+        {{- if .Values.volumePermissions.enabled }}
```

```suggestion
      {{- if or (and .Values.podSecurityContext.enabled .Values.volumePermissions.enabled .Values.persistence.enabled) (.Values.initContainers) }}
      initContainers:
        {{- if .Values.podSecurityContext.enabled .Values.volumePermissions.enabled .Values.persistence.enabled }}
```

Just to avoid creating the init container if persistence is disabled.